packages/cli/src/test-utils/multisigUtils.ts

```diff
@@ -0,0 +1,56 @@
+import { multiSigABI, proxyABI } from '@celo/abis'
+import { StrongAddress } from '@celo/base'
+import { ContractKit } from '@celo/contractkit'
+import { multiSigBytecode, proxyBytecode } from './constants'
+
+export async function createMultisig(
+  kit: ContractKit,
+  owners: StrongAddress[],
+  requiredSignatures: number,
+  requiredInternalSignatures: number
+): Promise<StrongAddress> {
+  const accounts = (await kit.web3.eth.getAccounts()) as StrongAddress[]
+  kit.defaultAccount = accounts[0]
+
+  // Deploy Proxy contract
+  const proxyDeploymentTx = await kit.sendTransaction({
+    data: proxyBytecode,
+  })
+  const { contractAddress: proxyAddress } = await proxyDeploymentTx.waitReceipt()
+  // Deploy MultiSig contract
+  const multisigDeploymentTx = await kit.sendTransaction({
+    data: multiSigBytecode,
+  })
+  const { contractAddress: multiSigAddress } = await multisigDeploymentTx.waitReceipt()
+
+  // Configure and initialize MultiSig
+  const initializerAbi = multiSigABI.find(
+    (abi) => abi.type === 'function' && abi.name === 'initialize'
+  )
+  const proxy = new kit.web3.eth.Contract(proxyABI as any, proxyAddress)
+  const baseFee = await kit.web3.eth.getBlock('latest').then((block: any) => block.baseFeePerGas)
+  const priorityFee = kit.web3.utils.toWei('2', 'gwei')
+  const initMethod = proxy.methods._setAndInitializeImplementation
+  const callData = kit.web3.eth.abi.encodeFunctionCall(initializerAbi as any, [
+    owners as any,
+    requiredSignatures as any,
+    requiredInternalSignatures as any,
+  ])
+  const initTx = initMethod(multiSigAddress, callData)
+  await initTx.send({
+    from: kit.defaultAccount,
+    gas: await initTx.estimateGas({ from: kit.defaultAccount }),
+    maxPriorityFeePerGas: priorityFee,
+    maxFeePerGas: (BigInt(baseFee) + BigInt(priorityFee)).toString(),
```

Comment:
I just remembered sometimes toString will return  scientific notation. i think toPrecision(10) is what is needed

Reply:
Ah interesting, I tried that, and it looks like `toPrecision(10)` only works on `Number` objects. Not for the `bigint` data type. 

Personally, I'm ok keeping this as is, because it's not a user-facing function, and only used internally for testing.